### DISCUSSION TITLE
Listing Review still pending

### DISCUSSION THREAD

**Discussion by wongzigii** [WalletConnect/walletconnect-monorepo#1808] [Developer Support]
I've submitted a explorer listing request. After two weeks, the request is still pending.
<img width="1639" alt="image" src="https://user-images.githubusercontent.com/7384288/209506871-1b254ec0-1b9a-4ec9-9625-d562ee914213.png">

**Comment by chenfan1227** [comment]
Mine is almost 2 months 😂

**Comment by bkrem** [comment]
Hi @wongzigii,

What's the name of the submission? The screenshot doesn't contain the name and/or UUID of your submission.

**Reply by wongzigii** [reply]
uuid=d8d2894b-84d0-4d72-be36-fcc68d2587a2 🙏

**Reply by bkrem** [reply]
Thanks @wongzigii, I'm not sure about that UUID (I can't locate an entry with it anywhere) but I think I found the intended submission (Loopring) from your (edited/removed?) comment.

Couple of issues:
* There seem to be two competing submissions from loopring accounts in pending state (see screenshot below)
* We already have imported legacy entries for Loopring in the Explorer (https://explorer.walletconnect.com/?search=loopring). I'm assuming we can remove these once your managed submission is approved?


<img width="484" alt="Screenshot 2023-01-06 at 16 04 41" src="https://user-images.githubusercontent.com/8663099/211039787-cd72776f-ccc0-4580-8054-045b185f9726.png">

**Reply by wongzigii** [reply]
Thanks @bkrem, the second is the legacy one, I've re-submitted one using the official wallet@loopring.io account.

**Reply by bkrem** [reply]
Thanks for context @wongzigii 👌 

The logo asset seems to be either missing or broken on the `wallet@loopring.io` pending submission (I just re-checked), so requested changes regarding that. 

We'll do a full review and hopefully approve once that's fixed 👍

**Reply by Morisander** [reply]
haha, sorry, i don't want to annoy anyone here, just on the search of how long this may take, so i can finally build my application in the loopring network and was happy to see you here ^^have a nice day^^
